src/modules/collection/selectors.spec.ts

```diff
@@ -3,12 +3,47 @@ import { getChainIdByNetwork } from 'decentraland-dapps/dist/lib/eth'
 import { RootState } from 'modules/common/types'
 import { SyncStatus } from 'modules/item/types'
 import { ThirdParty } from 'modules/thirdParty/types'
-import { getAuthorizedCollections, getStatusByCollectionId, hasViewAndEditRights } from './selectors'
+import { getAuthorizedCollections, getStatusByCollectionId, getUnsyncedCollectionError, hasViewAndEditRights } from './selectors'
 import { Collection } from './types'
+import { UNSYNCED_COLLECTION_ERROR_PREFIX } from './utils'
 
 jest.mock('decentraland-dapps/dist/lib/eth')
 const mockGetChainIdByNetwork = getChainIdByNetwork as jest.Mock
 
+describe('when getting the unsynced error message', () => {
+  it('should return null if the base error is null', () => {
+    const result = getUnsyncedCollectionError({
+      collection: {
+        error: null
+      }
+    } as any)
+
+    expect(result).toBeNull()
+  })
```

Comment:
Would you mind changing the tests so they fit with the [testing standards](https://www.notion.so/decentraland/Testing-standards-af326c6776d64d58857429583da41046#8c2d4a1bb0374a05853fc03083567004)?
That is, they should look like this:
```typescript
  let state: RootState
  describe('when the base error is null', () => {
    // We define an describe the state
    beforeEach(() => {
      result = {
        collection: {
          error: null
        }
      } as RootState
    })
  
    it('should return null', () => {
      const result = getUnsyncedCollectionError( as any)

      expect(result).toBeNull()
    })
  })
```